src/js/select2/a11y/a11yMulti.js

```diff
@@ -34,12 +35,14 @@ define([
       var $selection = this.selectionContainer();
       var formatted = this.display(selection, $selection);
 
-      if ('string' === typeof formatted) {
-        formatted = formatted.trim();
-      } 
-
       // Update selection summary (used for aria-describedby on search input)
-      this.$selectionSummary.append(formatted + ',');
+      if (typeof formatted === 'string') {
+        this.$selectionSummary.append(formatted.trim() + ',');
+      }
+
+      if (typeof formatted === 'object') {
+        this.$selectionSummary.append(formatted.text().trim() + ',');
```

Comment:
should we check if `formatted instanceof jQuery`? If I passed in a plain object here without a `text()` method or with `text()` method that did not return a string, it'd fall over here